### PR TITLE
feat: add additional mini portfolio events

### PR DESCRIPTION
**Pull request by lynnshaoyu**
https://www.notion.so/uniswaplabs/Mini-Portfolio-Main-Doc-product-spec-b5e26c04c05b40c4ac4f5c61b3248893?pvs=4#fb7b497088ce4ea7bcf73eec38079222

### CODE REVIEW

File: src/components/WalletDropdown/MiniPortfolio/NFT.tsx [hunk 1]
```diff
@@ -1,3 +1,6 @@
+import { useTrace } from '@uniswap/analytics'
+import { sendAnalyticsEvent } from '@uniswap/analytics'
```
Comment (grabbou) [inline src/components/WalletDropdown/MiniPortfolio/NFT.tsx:2]: nit: this should be a single import statement

File: src/components/WalletDropdown/MiniPortfolio/Pools/index.tsx [hunk 2]
```diff
@@ -99,43 +101,50 @@ function PositionListItem({ positionInfo }: { positionInfo: PositionInfo }) {
...
+      properties={{ chain_id: chainId, pool_token_0: pool.token0, pool_token_1: pool.token1 }}
+    >
+      <PortfolioRow
+        onClick={onClick}
+        left={<PortfolioLogo chainId={chainId} currencies={[pool.token0, pool.token1]} />}
+        title={
+          <Row>
             <ThemedText.SubHeader fontWeight={500}>
-              {formatNumber((liquidityValue ?? 0) + (feeValue ?? 0), NumberType.PortfolioBalance)}
+              {pool.token0.symbol} / {pool.token1?.symbol}
```
Comment (grabbou) [inline src/components/WalletDropdown/MiniPortfolio/Pools/index.tsx:116]: Is this change intended? From the PR description, looks like we're mostly concerned with the <TraceEvents /> component

File: src/nft/components/card/index.tsx [hunk 3]
```diff
@@ -80,6 +82,7 @@ export const NftCard = ({
       testId={testId}
       onClick={() => {
         if (bagExpanded) setBagExpanded({ bagExpanded: false })
+        if (sendAnalyticsEvent) sendAnalyticsEvent()
```
Comment (grabbou) [inline src/nft/components/card/index.tsx:85]: nit: this can be `sendAnalyticsEvent?.()`, same for bag expanded

Comment (lynnshaoyu) [reply src/nft/components/card/index.tsx:85]: ooh ty! thank you! I dont think bag expanded can be changed accordingly? setBagExpanded is different from bagExpanded

**Review by cartcrom — APPROVED** [review]
LGTM!
